### DISCUSSION TITLE
weewx-dwd

### DISCUSSION THREAD

**Discussion by JonFear** [Daveiano/weewx-wdc#113] [General]
I see that the Weewx-dwd (https://github.com/roe-dl/weewx-DWD) forecast plugin has been updated to include various different models in addition to just the dwd forecast. Will the changes in the plugin still be usable under the latest version? I could not get it working previously but that is just down to me...

I would love to be able to use the ecmwf on my pages...

Sorry, more work....!

Jon

**Comment by Daveiano** [comment]
Thank you for your participation in the project!

And thanks for the reminder, I was not aware of the recent changes in weewx-dwd. 

> I could not get it working previously but that is just down to me...

Sorry for that, I wrote the [wiki page about weewx-dwd](https://github.com/Daveiano/weewx-wdc/wiki/Support-for-weewx-DWD) in german because at this time it provided only the dwd model and I thought only users would be german users... 

> Will the changes in the plugin still be usable under the latest version?

I have not tested it myself but it should work, it would be very helpful if you could test it out, maybe.

**Comment by JonFear** [comment]
I think we can close out this one as well...
